postreise/plot/plot_powerflow_snapshot.py

```diff
@@ -118,7 +117,7 @@ def plot_powerflow_snapshot(
     :param str solar_color: color to plot solar generation.
     :param str wind_color: color to plot wind generation.
     :param str demand_color: color to plot demand.
-    :param tuple(int, int) figsize: size of the bokeh figure (in pixels).
+    :param tuple figsize: size of the bokeh figure (in pixels).
```

Comment:
Is this not kosher?

Reply:
I have got for principle to only indicate the type there. Further information should go in the description. When we have a `dict`, we don't indicate the type of the keys and values.